Index: src/main/java/com/dongdong/controller/A.java
===================================================================
diff --git a/src/main/java/com/dongdong/controller/A.java b/src/main/java/com/dongdong/controller/A.java
deleted file mode 100644
--- a/src/main/java/com/dongdong/controller/A.java	(revision 2d13971c0d77f565c98d31f48723b759e76dc9c1)
+++ /dev/null	(revision 2d13971c0d77f565c98d31f48723b759e76dc9c1)
@@ -1,4 +0,0 @@
-package com.dongdong.controller;
-
-public class A {
-}
